src/standalone/Snow/boundary_fluxes.jl

```diff
@@ -0,0 +1,127 @@
+"""
+    AbstractSnowBC <: ClimaLand.AbstractBC
+
+An abstract type for boundary conditions for the snow model.
+"""
+abstract type AbstractSnowBC <: ClimaLand.AbstractBC end
+
+"""
+    AtmosDrivenSnowBC{
+        A <: AbstractAtmosphericDrivers,
+        B <: AbstractRadiativeDrivers,
+        C::Tuple
+    } <: AbstractSnowBC
+
+A struct used to specify the snow fluxes, referred
+to as ``boundary conditions", at the surface and
+bottom of the snowpack, for water and energy.
+
+These fluxes include turbulent surface fluxes
+computed with Monin-Obukhov theory, and radiative fluxes.
+$(DocStringExtensions.FIELDS)
+"""
+struct AtmosDrivenSnowBC{
+    A <: AbstractAtmosphericDrivers,
+    B <: AbstractRadiativeDrivers,
+    C <: Tuple,
+} <: AbstractSnowBC
+    "The atmospheric conditions driving the model"
+    atmos::A
+    "The radiative fluxes driving the model"
+    radiation::B
+    "Prognostic land components present"
+    prognostic_land_components::C
+end
+
+function AtmosDrivenSnowBC(
+    atmos,
+    radiation;
+    prognostic_land_components = (:snow,),
+)
+    args = (atmos, radiation, prognostic_land_components)
+    return AtmosDrivenSnowBC(args...)
+end
+
+
+"""
+    snow_boundary_fluxes!(bc::AtmosDrivenSnowBC, model::SnowModel, Y, p, t)
+
+Updates in place various volumetric water flux (m/s) and energy
+flux (W/m^2) terms for the snow model:
+
+- p.snow.turbulent fluxes (latent, sensible, and evaporative fluxes)
+- p.snow.R_n (radiative fluxes)
+- p.snow.total_water_flux
+- p.snow.total_energy_flux
+
+The two latter fluxes include fluxes due to melt and precipitation,
+but these are computed in the update_aux! function currently.
```

Comment:
I'm a little confused by this sentence - are they both computed and updated in `update_aux!`, or are they updated here?